README.md

```diff
@@ -1 +1,111 @@
-# Seafood-capstone-project
\ No newline at end of file
+<div align="center">
+
+  <img src="./src/Asset/logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h1><b>Seafood Capstone</b></h1>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+## About the Project
+
+> This project is a capstone project for Microverse, where you can see meals that we got from the API, on this website you are able to like the meals, comment the meals, and add reservation for the meals.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Webpack
+- Jest
+- API
+
+### Key Features
+
+- Getting meal cards from the API
+- Liking the meals with the counter
+- Adding comments on it
+- Adding reservations for the meals
+
+## Live Demo
+
+[Live Demo](https://zeff96.github.io/Seafood-capstone-project/dist/)
+
+## Getting Started
+
+### Clone this [repository](https://github.com/zeff96/Seafood-capstone-project) to your desired folder.
+
```

Comment:
- [ ] Kindly make sure you add the `## Setup` sub-section with appropriate information such as how to clone your repo, access the cloned directory,... This will help non technical users on how to proceed and to make your documentation more not only descriptive, but also professional. For an example, just refer this **[template](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)**

Reply:
Added changes requested